auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -362,10 +363,39 @@ def delete_affiliation_invitation(invitation_id):
         else:
             invitation.delete()
 
+    @staticmethod
+    def _filter_request_invites_role_based(affiliation_invitation_models: List[AffiliationInvitationModel],
+                                           org_id: int,
+                                           user: UserService = None) -> List[AffiliationInvitationModel]:
```

Comment:
It doesn't look like there is an explicit use of the user here, and the comment indicates current user.

Can we remove this param?

Reply:
Yeah, was passing it previously from another method, but after bit of refactor its not being passed anymore. And agreed, if its not explicitly used, remove!
Thanks for catching it.